Tests/Marketplace/marketplace_services.py

```diff
@@ -70,7 +70,7 @@ class Metadata(object):
     DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
     XSOAR_SUPPORT = "xsoar"
     XSOAR_SUPPORT_URL = "https://www.paloaltonetworks.com/cortex"  # disable-secrets-detection
-    XSOAR_AUTHOR = "Cortex XSOAR"
+    XSOAR_AUTHOR = "Palo Alto Networks"
```

Comment:
@yuvalbenshalom  Pay attention also for this changes, you cannot see it diff due to large number of changed packs